pkg/api/v1alpha1/cluster_types.go

```diff
@@ -307,6 +307,8 @@ type ControlPlaneConfiguration struct {
 	CertSANs []string `json:"certSans,omitempty"`
 	// MachineHealthCheck is a control-plane level override for the timeouts and maxUnhealthy specified in the top-level MHC configuration. If not configured, the defaults in the top-level MHC configuration are used.
 	MachineHealthCheck *MachineHealthCheck `json:"machineHealthCheck,omitempty"`
+	// APIServerExtraArgs defines the flags to configure for the API server.
+	APIServerExtraArgs map[string]string `json:"apiServerExtraArgs,omitempty"`
```

Comment:
```suggestion
	ApiServerExtraArgs map[string]string `json:"apiServerExtraArgs,omitempty"`
```

Should we change to this casing everywhere in the PR?

Reply:
`golangci-lint` check suggests all capital letters for [initialisms](https://go.dev/wiki/CodeReviewComments#:~:text=return%0A%7D%0A//%20use%20x-,Initialisms,-Words%20in%20names). I can update it to API everywhere for consistency